packages/wallet/src/test-utils/launchNode.ts

```diff
@@ -106,27 +136,41 @@ export const launchNodeAndGetWallets = async ({
   launchNodeOptions?: Partial<LaunchNodeOptions>;
   walletCount?: number;
 } = {}) => {
-  // Write a temporary chain configuration file.
-  await fs.writeFile('.chainConfig.json', JSON.stringify(defaultChainConfig), 'utf8');
+  const osTempDir = os.tmpdir();
+
+  if (!fsSync.existsSync(osTempDir)) {
+    fsSync.mkdirSync(osTempDir);
+  }
 
-  // Create a temp db directory.
-  await fs.mkdir('.fuel-core-db', { recursive: true });
+  const subDirName = '.fuels-ts'; // Change to your desired subfolder name
+  const subDirPath = path.join(osTempDir, subDirName);
+
+  if (!fsSync.existsSync(subDirPath)) {
+    fsSync.mkdirSync(subDirPath);
```

Comment:
```suggestion
    fsSync.mkdirSync(subDirPath, { recursive: true });
```

You can improve this by just informing node.js to do it recursively. In this way, node.js is going to create all the folders even if the path does not exist. If exists is going to just ignore it.

Also in this way no need to the first checks line 141..143

Reply:
Great catch @luizstacio